src/api/status/public/js/posts-count.js

```diff
@@ -0,0 +1,18 @@
+const totalPostsEl = document.getElementById('totalPosts');
+const fetchPostsCount = () =>
+  fetch('https://dev.api.telescope.cdot.systems/v1/posts').then((res) =>
+    res.headers.get('x-total-count')
+  );
+
+const getPostsCount = async () => {
+  try {
+    const total = await fetchPostsCount();
+    totalPostsEl.innerText = total ?? 'N/A';
```

Comment:
We should leave innerText blank or N/A because I need to know to make mine the same.

Reply:
> We should leave innerText blank or N/A because I need to know to make mine the same.

I think it should be N/A to make it clearer